### PR TITLE
Added table filtering

### PR DESCRIPTION
**Pull request by yogs22**

### CODE REVIEW

File: src/components/Home/Home.js [hunk 1]
```diff
@@ -85,11 +90,32 @@ class Home extends Component {
     }, 5000);
   }
 
+  filterProject(event) {
+    var updatedProject = this.state.initialProjects;
+    updatedProject = updatedProject.filter(function(item) {
```
Comment (TheWillG) [inline src/components/Home/Home.js]: Please use ES6 arrow functions and change name to filteredProjects.

File: src/components/Home/Home.js [hunk 2]
```diff
@@ -85,11 +90,32 @@ class Home extends Component {
...
+      const query = event.target.value.toLowerCase();
+
+      return(
+        item.name.toLowerCase().indexOf(query) >= 0 ||
```
Comment (TheWillG) [inline src/components/Home/Home.js]: Please combine name, description, and tech into one string using template literals and search the index of this. Such as 
const tableContent=`${item.name}${item.description}${item.tech}`;
return tableContent.toLowercase.....

File: src/components/Home/Home.js [hunk 3]
```diff
@@ -50,22 +51,26 @@ class Home extends Component {
     this.state = {
       modalOpen: false,
       snackbarOpen: false,
+      initialProjects: [],
       projects: [],
       snackbarText: '',
+      query: '',
```
Comment (TheWillG) [inline src/components/Home/Home.js]: Is this used?

File: src/components/Home/Home.js [hunk 4]
```diff
@@ -50,22 +51,26 @@ class Home extends Component {
...
     };
     this.addProject = this.addProject.bind(this);
     this.modalClosed = this.modalClosed.bind(this);
     this.fetchProjects = this.fetchProjects.bind(this);
     this.renderSnackbar = this.renderSnackbar.bind(this);
+    this.filterProject = this.filterProject.bind(this);
   }
 
-  async componentDidMount() {
+  async componentWillMount() {
```
Comment (TheWillG) [inline src/components/Home/Home.js]: Please keep componentDidMount. We want the component to render once before fetching the data.

**Comment by yogs22** [comment]
Okay.. i'm just beginner in react development

**Comment by yogs22** [comment]
Okay.. i'm just beginner in react development

**Comment by TheWillG** [comment]
@yogs22 No problem. Let me know if you have any questions with my corrections above.

File: src/components/Home/Home.js (hunk 2 above)
Comment (yogs22) [reply src/components/Home/Home.js]: hi, i can't use literal in this case ?
can you show me other ways ?

**Comment by yogs22** [comment]
i've done this issue

**Comment by yogs22** [comment]
i've done this issue @TheWillG

File: src/components/Home/Home.js (hunk 1 above)
Comment (TheWillG) [reply src/components/Home/Home.js]: @yogs22 Please use ES6 arrow function (item) => {}

File: src/components/Home/Home.js [hunk 5]
```diff
@@ -85,11 +88,30 @@ class Home extends Component {
     }, 5000);
   }
 
+  filteredProject = (event) => {
```
Comment (TheWillG) [inline src/components/Home/Home.js]: @yogs22 Please use `filteredProjects` with an `s` at the end only for this function name

**Comment by TheWillG** [comment]
@yogs22 please see my latest comments

**Comment by TheWillG** [comment]
@yogs22 please fast forward with master branch and resolve conflicts

**Pull request merged**